prisma/migrations/20221107035551_populate_label_table/migration.sql

```diff
@@ -0,0 +1,4 @@
+-- This is a fatto a mano migration to populate the Label table with the first, experimental label value.
+
+INSERT INTO Label (externalId, name, createdBy, updatedAt) VALUES ('794ad945-e385-4820-b350-799a40ea5868', 'region-east-africa', 'backend-product-squad', CURRENT_TIMESTAMP(0));
```

Comment:
doesn't the `seed` script already create some labels and collection-label associations and that can be executed with prisma migrate? This could be added to the seed